src/patternfly/components/Form/docs/code.md

```diff
@@ -7,7 +7,7 @@
 | `required aria-required="true"` | `<input>`, `<select>`, `<textarea>` | Required fields must include these attributes. |
 | `id="{helper_text_id}"` | `.pf-c-form__helper-text` | Form fields with related `.pf-c-form__helper-text` require this attribute. Usage `<p class="pf-c-form__helper-text" id="{helper_text_id}" ...>`.  |
 | `aria-describedby="{helper_text_id}"` | `<input>`, `<select>`, `<textarea>` | Form fields with related `.pf-c-form__helper-text` require this attribute. Usage `<input aria-describedby="{helper_text_id}" ...>`.  |
```

Comment:
Is `...` in `<input aria-describedby="{helper_text_id}" ...>` intentional? I assume it denotes 'other' attributes, but idk if it's necessary.

Reply:
good question, that was from the work that was already done on forms ;) Happy to remove.